src/Components/CreateImageWizardV2/steps/Packages/Packages.tsx

```diff
@@ -886,6 +891,17 @@ const Packages = () => {
                     onChange={handleSearch}
                   />
                 </InputGroupItem>
+                {searchTerm && (
+                  <InputGroupItem>
+                    <Button
```

Comment:
nitpick: it would be nice to add ouia-id for this behavior

Reply:
Great point, I've added one for the button component